tests/objects/test_bitmask_numpy.py

```diff
@@ -0,0 +1,96 @@
+    rle = _string_to_rle(encoded_str)
+    mask = _rle_to_mask(rle, size=resolution)
+    assert len(mask) == resolution
+
+
+def test_mask_roundtrip():
+    orig_mask = np.zeros((2000, 3000), dtype=bool)
+    orig_mask[:1000, 1000:] = 1
+    orig_mask[1000:, :1000] = 1
+
+    mask_coordinates = BitmaskCoordinates(orig_mask)
+
+    new_mask = mask_coordinates.to_numpy_array()
+    assert np.allclose(orig_mask, new_mask)
+
+    new_mask_using_constructor = np.array(mask_coordinates)
+    assert np.allclose(orig_mask, new_mask_using_constructor)
+
+
+def test_mask_rle_mask_roundtrip():
+    orig_mask = np.zeros((2000, 3000), dtype=bool)
+    orig_mask[:1000, 1000:] = 1
+    orig_mask[1000:, :1000] = 1
+
+    mask_coordinates = BitmaskCoordinates(orig_mask)
+    mask_dict = mask_coordinates.to_dict()
+
+    new_mask = BitmaskCoordinates.from_dict(mask_dict).to_numpy_array()
+    assert np.allclose(orig_mask, new_mask)
+
+
+def test_transpose_bytearray_python():
+    mask = b"\00\00\00\00\00\01\01\00\00\00\00\00"
+    shape = (3, 4)
+    expected = b"\x00\x00\x00\x00\x01\x00\x00\x01\x00\x00\x00\x00"
+    ans = transpose_bytearray(mask, shape)
+    assert ans == expected
+
+
+def test_transpose_bytearray_numpy():
```

Comment:
Repeated both tests (which themselves are a repeat of each other). Probably here you should check both implementations and delete other tests.